lib/galaxy/webapps/galaxy/controllers/workflow.py

```diff
@@ -518,14 +496,26 @@ def copy(self, trans, id, save_as_name=None):
         return_url = url_for('/') + 'workflow?status=done&message=%s' % escape(message)
         trans.response.send_redirect(return_url)
 
-    @web.expose
-    @web.require_login("create workflows")
-    def create(self, trans, workflow_name=None, workflow_annotation=""):
-        """
-        Create a new stored workflow with name `workflow_name`.
-        """
-        user = trans.get_user()
-        if workflow_name is not None:
+    @web.expose_api
+    def create(self, trans, payload=None, **kwd):
+        if trans.request.method == 'GET':
+            return {
+                'title'  : 'Create Workflow',
+                'inputs' : [{
+                    'name'  : 'workflow_name',
```

Comment:
thanks @guerler 

I want to share one use case not specifically to this pull request but in general related to workflows. 
When a user creates a workflow, "Unnamed workflow" appears as the default workflow name. Let's say it is saved. If she/ he creates another workflow, the default workflow name appears as "Unnamed workflow" again (even though there is already a workflow present with this name) and the application allows to have the same workflow name. Thanks!

![unnamed_workflow](https://user-images.githubusercontent.com/3022518/32778118-fc2336e6-c938-11e7-9d7d-39f4c94f38c1.png)